tests/helpers.py

```diff
@@ -706,3 +707,122 @@ def get_worker_nodes():
     nodes = ocp_node_obj.get(selector=label).get('items')
     worker_nodes_list = [node.get('metadata').get('name') for node in nodes]
     return worker_nodes_list
+
+
+def create_dc_pod(
```

Comment:
please try to utilize the existing create_pod() function we have in helpers, to avoid code duplication

Reply:
done